css/properties/font-synthesis-style.json

```
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "properties": {
+      "font-synthesis-style": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-fonts/#font-synthesis-style",
+          "support": {
+            "chrome": {
+              "version_added": "97"
+            },
+            "chrome_android": {
+              "version_added": "97"
+            },
+            "edge": {
+              "version_added": "97"
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "<a href='https://bugzil.la/1724892'>bug 1724892</a>."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "<a href='https://bugzil.la/1724892'>bug 1724892</a>."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "83"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false,
+              "notes": "<a href='https://webkit.org/b/232009'>bug 232009</a>."
+            },
+            "safari_ios": {
+              "version_added": false,
+              "notes": "<a href='https://webkit.org/b/232009'>bug 232009</a>."
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "97"
+            }
+          },
+          "status": {
+            "experimental": false,
```

Until there are multiple implementations, this should be experimental. See the [_Choosing an experimental status_ data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status) for details.

```suggestion
            "experimental": true,
```